docs/contents/proposals/secrets/secrets.md

```diff
@@ -37,24 +115,47 @@ secretRefs:
     - action: CreateDBCluster
 ```
 
-These API specific YAML files will be available for users to download and save into their directories of choice.
-Let us assume the user Alice wants to use the Amazon RDS API to create a new DB Instance. Alice must create a Kubernetes Secret in the desired namespace and save the RDS YAML file in the directory she would like the DBInstance to reside in. Then, she simply calls the kubectl apply command, passes in her specification, and waits for a response from the Kubernetes API server.
+### How will the user interact with the proposed solution?
+Let us assume the user Alice wants to use the Amazon RDS API to create a new DB Instance. Alice must first create a Kubernetes Secret object and specify a key within the Secret that will store her RDS DB Instance master user password. Alice will reference that Secret and key within her RDS DB Instance custom resource manifest. Alice then simply calls the [kubectl apply command](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#kubectl-apply), passes in her specification, and waits for a response from the [Kubernetes API](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) server. 
```

Comment:
Here, after the line "Alice will reference that Secret and key within her RDS DB Instance custom resource manifest", that's where you should put the YAML content from above.